global.json

```diff
@@ -1,10 +1,10 @@
 {
   "sdk": {
-    "version": "5.0.400",
+    "version": "6.0.0",
```

Comment:
You need to reference an actual, specific version in `global.json`. [Steal the `global.json` from Autofac.Extensions.DepdencyInjection, where we already target .NET 6.](https://github.com/autofac/Autofac.Extensions.DependencyInjection/blob/develop/global.json)

Reply:
Done